accounts-db/src/accounts_db.rs

```diff
@@ -4004,7 +4003,6 @@ impl AccountsDb {
                         // not exist in the re-written slot. Unref it to keep the index consistent with
                         // rewriting the storage entries.
                         unrefed_pubkeys.push(pubkey);
-                        result = AccountsIndexScanResult::Unref;
```

Comment:
well the plan is that we don't bring them into memory. if they are on disk, we will return None. If they are in memory, we'll look at them here. Since they're in memory, they'll remain in memory. If we are going to unref them, that means they have a ref from someone else, too, so they'll remain in memory anyway.

Reply:
Not necessarily. 
These are candidates. We may decide not pack them later on.